From 119f58c705fcd9749b847c1e0e64049f816da30a Mon Sep 17 00:00:00 2001
From: anonymous <anonymous@inventec.com>
Date: Thu, 13 Aug 2020 16:10:41 -0400
Subject: [PATCH 1/4] Subject: [PATCH] Subject: [PATCH][Entity-Manager] Add a
 writable dbus interface "MonitorSettings"

- Add a writable dbus interface "MonitorSettings" for run-time configuring sensor.
---
 src/EntityManager.cpp | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/src/EntityManager.cpp b/src/EntityManager.cpp
index 84d68c6..cdf1fef 100644
--- a/src/EntityManager.cpp
+++ b/src/EntityManager.cpp
@@ -77,8 +77,9 @@ const static boost::container::flat_map<const char*, probe_type_codes, cmp_str>
                  {"FOUND", probe_type_codes::FOUND},
                  {"MATCH_ONE", probe_type_codes::MATCH_ONE}}};
 
-static constexpr std::array<const char*, 6> settableInterfaces = {
-    "FanProfile", "Pid", "Pid.Zone", "Stepwise", "Thresholds", "Polling"};
+static constexpr std::array<const char*, 7> settableInterfaces = {
+    "FanProfile", "Pid", "Pid.Zone", "Stepwise", "Thresholds", "Polling",
+    "MonitorSettings"};
 using JsonVariantType =
     std::variant<std::vector<std::string>, std::vector<double>, std::string,
                  int64_t, uint64_t, double, int32_t, uint32_t, int16_t,
-- 
2.7.4

